Mflowgen fix

steveri opened the pull request:
ADK setup for CI was outdated/incorrect. Previously, it assumed that, if the adk was missing, it could be found at a backup location `/sim/buildkite-agent/adks`. This works on the TSMC machine, but not the GF machine. I cannot access the GF machine, because of licensing, but I know at least one copy of the adks can be found at  `/sim/buildkite-agent/mflowgen.master/adks`, so I changed the script to use that for the GF adk backup.

If anyone knows of a better place to find `gf12-adk` on the GF machine, let me know!

alexcarsello reviewed (changes requested):
On the GF machine, if Buildkite doesn't already have the ADK cloned, it should just clone it by doing `git clone /sim/repos/gf12-adk`

steveri commented:
> On the GF machine, if Buildkite doesn't already have the ADK cloned, it should just clone it by doing `git clone /sim/repos/gf12-adk`

Thanks very much Alex, I have updated the code accordingly.

steveri commented:
> > On the GF machine, if Buildkite doesn't already have the ADK cloned, it should just clone it by doing `git clone /sim/repos/gf12-adk`
> 
> Thanks very much Alex, I have updated the code accordingly.

Also: as you suggested, I changed the mflowgen pipeline so that it flags a gold-RTL error but does not fail the pipeline. Examples:
https://buildkite.com/tapeout-aha/mflowgen/builds/7119
https://buildkite.com/tapeout-aha/mflowgen/builds/7120
